packages/create-app/templates/default-app/packages/app/src/index.tsx

```diff
@@ -1,6 +1,8 @@
 import '@backstage/cli/asset-types';
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { createRoot } from 'react-dom/client'
```

Comment:
You'll need to update `packages/create-app/templates/default-app/packages/app/package.json` as well, in the same way as the example app